src/snowflake/cli/plugins/spcs/services/manager.py

```diff
@@ -91,3 +92,53 @@ def upgrade_spec(self, service_name: str, spec_path: Path):
 
     def list_endpoints(self, service_name: str) -> SnowflakeCursor:
         return self._execute_schema_query(f"show endpoints in service {service_name}")
+
+    def set_property(
+        self,
+        service_name: str,
+        min_instances: Optional[int],
+        max_instances: Optional[int],
+        query_warehouse: Optional[str],
+        auto_resume: Optional[bool],
+        comment: Optional[str],
+    ):
+        property_pairs = [
+            ("min_instances", min_instances),
+            ("max_instances", max_instances),
+            ("query_warehouse", query_warehouse),
+            ("auto_resume", auto_resume),
+            ("comment", comment),
+        ]
+        if all([value is None for property_name, value in property_pairs]):
+            raise NoPropertiesProvidedError(
+                f"No properties specified for service '{service_name}'. Please provide at least one property to set."
+            )
+        query: list[str] = [f"alter service {service_name} set"]
+        for property_name, value in property_pairs:
+            if value is not None:
+                query.append(f"{property_name} = {value}")
+        return self._execute_schema_query(strip_empty_lines(query))
+
+    def unset_property(
+        self,
+        service_name: str,
+        min_instances: bool,
+        max_instances: bool,
+        query_warehouse: bool,
+        auto_resume: bool,
+        comment: bool,
+    ):
+        property_pairs = [
+            ("min_instances", min_instances),
+            ("max_instances", max_instances),
+            ("query_warehouse", query_warehouse),
+            ("auto_resume", auto_resume),
+            ("comment", comment),
+        ]
+        if not any([value for property_name, value in property_pairs]):
```

Comment:
It's a bit tricky. Could you add a comment to indicate if the property is not specified, the value will be false?